File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/DatePoint.java
package com.kendoui.spring.models;


public class DatePoint {
    private String date;
    private double value;
    
    public DatePoint(double value, String date) {
        setValue(value);
        setDate(date);
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public double getValue() {
        return value;
    }

    public void setValue(double value) {
        this.value = value;
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/DatePoint.cs
using System;

namespace Kendo.Mvc.Examples.Models
{
    public class DatePoint
    {
        public DatePoint(double value, DateTime date)
        {
            Value = value;
            Date = date;
        }

        public DateTime Date { get; set; }
        public double Value { get; set; }
    }
}
